### PR TITLE
Update kubernetes components to head

### PR DESCRIPTION
**Pull request by dlorenc**
Not sure what changed between the two commits, but it now compiles and passes tests.

It took a few int/int32 swaps and a couple other minor changes.

#70

### CODE REVIEW

**Comment by dlorenc** [comment]
cc @luxas

**Comment by ethernetdan** [comment]
@dlorenc did you use the godeps.sh script to update this? It doesn't look like the `vendor/golang.org/x/net/trace/trace.go` patch got applied

**Comment by dlorenc** [comment]
> @dlorenc did you use the godeps.sh script to update this? It doesn't look like the vendor/golang.org/x/net/trace/trace.go patch got applied

Ah, no I didn't. Everything appears to be working though, maybe we don't need the patch anymore? How did you repro the panic before?

**Comment by ethernetdan** [comment]
Great! yeah looks like its not necessary anymore

File: minikube [hunk 1]
```diff
@@ -0,0 +1 @@
+/usr/local/google/home/dlorenc/kgo/src/k8s.io/minikube
```
Comment (luxas) [inline minikube]: Remove this file

**Comment by luxas** [comment]
@dlorenc Please also remove the `godep.sh` patch if it isn't required

**Comment by luxas** [comment]
LGTM, let's merge this beast!

**Pull request merged**